### ISSUE TITLE
ClojureScript: Invalid :refer, var instaparse.core/defparser does not exist

### ISSUE THREAD

**Issue by davidpham87** [green-coder/html-to-hiccup#3]
Hello Vincent,

I tried to use your library on ClojureScript, but I think you might need a different import statement for ClojureScript. According to the README from the instaparse library:

```clojure
;; Clojure
(:require [instaparse.core :as insta :refer [defparser]])
;; ClojureScript
(:require [instaparse.core :as insta :refer-macros [defparser]])
```

So I guess this should correct the issue in the declaration of the namespace:

```clojure
(ns taipei-404.html
  (:require [clojure.string :as str]
            [clojure.edn :as edn]
            #?(:clj [instaparse.core :as insta :refer [defparser]]
               :cljs [instaparse.core :as insta :refer-macros [defparser]])))
```

Best regards,
David

**Comment by green-coder** [comment]
Thank you for the report. I haven't had any problem while running the tests using Kaocha so I was assuming that my require was correct.

Please let me know which Clojurescript compiling tools you are using as well as the versions for Clojurescript and Clojure, so I can try to reproduce the problem locally.

**Comment by davidpham87** [comment]
I am using shadow-cljs (latest version). Maybe it is because of the version of ClojureScript, I might not use the latest version.

**Comment by green-coder** [comment]
I am able to reproduce it locally and you were right about the problem and the fix. Thank you.

**Comment by green-coder** [comment]
@davidpham87  It's fixed in the `v0.1.4` release

**Comment by davidpham87** [comment]
Thanks a lot for the swift resolution! :)
